core/src/test/java/io/hyperfoil/core/impl/statistics/StatisticsStoreTest.java

```diff
@@ -77,39 +76,101 @@ public void allJson_empty_issue45() {
 
    @Test
    public void allJson() {
+
       ErgonomicsBuilder ergonomicsBuilder = new ErgonomicsBuilder();
       ergonomicsBuilder.repeatCookies(true).userAgentFromSession(true);
       Benchmark benchmark = new BenchmarkBuilder("originalSource", BenchmarkData.EMPTY)
             .name("benchmarkName")
             .http().host("localhost").endHttp()
-            .addPhase("phaseName/001/test").always(1)
+
+            .addPhase("ramp/001/one").always(1)
```

Comment:
I prefer to improve the readability of such long fluent configuration by increasing indentation when you get into a child structure. To avoid reducing the indentation you can wrap it with comments `// @formatter:off` and `// @formatter:on`